docs/content/community/presentations/index.md

```diff
@@ -0,0 +1,26 @@
+---
+type: docs
+title: "Give a Radius presentation"
+linkTitle: "Presentations"
+description: "Learn how to give a Radius presentation, with downloadable slides and assets"
+weight: 400
+---
+
+We welcome community members giving presentations on Radius and spreading the word about all the awesome Radius features!
+
+## Giving a Radius presentation
+
+1. Begin by downloading the [Radius Presentation Deck](https://github.com/radius-project/docs/raw/{{< param github_branch >}}/presentation-pack/Radius.potx). This contains slides needed to give a Radius presentation.
+1. Refer to the Radius [presentation pack directory](https://github.com/radius-project/docs/tree/{{< param github_branch >}}/presentation-pack) for any diagrams or images you need to create.
+1. Next, review the docs to make sure you understand the [concepts]({{< ref concepts >}}).
```

Comment:
We need to cleanup the concept page needs as some old pages are showing up in this root view.

Reply:
Good catch. Could you open a GH Issue for this please?

Reply:
https://github.com/radius-project/docs/issues/1022